climpred/bootstrap.py

```diff
@@ -340,12 +342,22 @@ def bootstrap_compute(
         results = results.drop(res_drop)
         ci = ci.drop(ci_drop)
     results = xr.concat([results, ci], 'results')
-    results['results'] = [
-        'skill',
-        'p',
-        'low_ci',
-        'high_ci',
-    ]  # add ci levels in metadata
+    results['results'] = ['skill', 'p', 'low_ci', 'high_ci']
+    metadata_dict = dict()
+    metadata_dict['confidence interval levels'] = str(ci_high) + ' ' + str(ci_low)
+    metadata_dict['bootstrap iterations'] = bootstrap
+    metadata_dict[
+        'p'
+    ] = 'probability that initialized forecast performs \
+                          better than reference forecast'
+    results = assign_climpred_compute_to_attrs(
+        results,
+        hind,
+        metric=metric,
+        comparison=comparison,
+        function_name=inspect.stack()[0][3],
```

Comment:
This is the main function he's adding that adds attributes to the output object. But I agree that a comment line should be added here. @ahuang11, do you have any ideas on how to clean this method up? I think it will balloon compute functions to add all the dictionary + attrs function to the end. Maybe a separate private function in the module to handle this, just to keep things cleaner. I was thinking a decorator, but as @aaronspring pointed out, it relies on a lot of arguments/variables that are generated within the function.

Reply:
I meant to say I was confused about [0][3] on inspect.stack()

Someday, when we we're going to 2.0, we may want to think about a consistent way to name the functions/arguments. I feel like they're very inconsistent: e.g. `metric` vs `function_name` (all three are names, I think but one has `_name` suffix)

Reply:
Similar to how metpy deprecated get_windspeed -> windspeed
https://unidata.github.io/MetPy/latest/api/generated/metpy.calc.get_wind_speed.html

Reply:
`[0][3] on inspect.stack()` gets the functions name when called inside the function. like `function.__name__` I use this in args: should I just put the comment behind that function line @ahuang11 ?